src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,16 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        for (int i = 0; i < records.length; i++) {
+            if (email.equals(records[i].substring(0, records[i].indexOf(":")))) {
```

Comment:
instead of using substring + indexOf twice, try to use String[] yourVariable = records.split("symbol to split here")

Reply:
done